https://github.com/AdaCore/gprbuild/commit/b84df5ca242f0a144dda7f3b26c1c1184a796976

From 6421e350274b3018a26bd058b1c90d033b053f71 Mon Sep 17 00:00:00 2001
From: Eric Botcazou <ebotcazou@adacore.com>
Date: Wed, 29 Oct 2025 22:10:58 +0100
Subject: [PATCH] Fix another case of generic formal parameter used outside of
 instance

The modified compiler gives:

gpr-compilation-process.adb:44:29: error: operator for type "String" is not declared in "Env_Maps"

Issue: eng/toolchain/gnat#1597
---
 gpr/src/gpr-compilation-process.adb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gpr/src/gpr-compilation-process.adb b/gpr/src/gpr-compilation-process.adb
index ed76e037..87f9d37a 100644
--- a/gpr/src/gpr-compilation-process.adb
+++ b/gpr/src/gpr-compilation-process.adb
@@ -41,7 +41,7 @@ package body GPR.Compilation.Process is
    --  A set of key=value
 
    package Prj_Maps is new Containers.Indefinite_Ordered_Maps
-     (String, Env_Maps.Map, Env_Maps."<", Env_Maps."=");
+     (String, Env_Maps.Map, "<", Env_Maps."=");
    --  A set of project+language=map
 
    function "<" (Left, Right : Id) return Boolean is

